packages/EasyCore/src/Doctrine/Type/DateTimeMicrosecondsType.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Type;
+
+use DateTime;
+use DateTimeInterface;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+
+final class DateTimeMicrosecondsType extends Type
+{
+    public const TYPE_NAME = 'datetime';
```

Comment:
Please let's make sure ALL class constants are typed using a PHPDoc bloc. https://tomasvotruba.com/blog/2020/06/22/why-class-constants-should-be-typed/

Reply:
Done. Frankly saying, I'm a bit confused which rules to follow in this package because the rules differ from our projects. So I tend to rely on automatic code style checking.

Reply:
Yes I understand and the plan is to make everything consistent moving forward, it's coming 😄